nix/stack-shell.nix

```diff
@@ -2,7 +2,7 @@
 
 let
   sources = (import ./sources.nix);
-  pkgs = import sources.nixpkgs {};
+  pkgs = import sources.nixpkgs { };
```

Comment:
It would be better to get rid of niv and `sources` if we're fully switching to flakes

Reply:
done!